modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+You can add or update a CLI plugin to the {cli-manager} using the Form View or YAML View.
+
+[NOTE]
+====
+If you use the form view, add the information for the CLI plugin into the required fields, which are *Name*, *Platforms (bin, from)*, *Files*, *Image*, *ImagePullSecret*, *Platform*, *Short description*, *Version*, and *Homepage*. Then click *Create* to add the CLI plugin.
+====
+
+.Prerequisites
+
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* The {cli-manager} is installed.
+* The CliManager operand has been created.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name>
+spec:
+  shortDescription: short description of the <plugin_name>
+  description: long description of the <plugin_name>
+  version: <version> <1>
+  caveats: <2>
+  homepage: <plugin_homepage>
+  platforms:
+  - platform: <platform> <3>
+    image: <plugin_image>
+    imagePullSecret: <4>
+    files:
+    - from: <plugin_file_path>
+----
+<1> Version must be in v0.0.0 format.
+<2> This field is optional.
```

Comment:
Can you give some detail on what this field is for? And is it just text information or something?

And you can start it with the prefix "Optional: <description>"